frontend/src/components/pages/CreateReview/CreateReview.tsx

```diff
@@ -150,11 +168,11 @@ const CreateReview = ({ id }: CreateReviewProps): React.ReactElement => {
   };
 
   /**
-   * Function to be called when the review is published.
+   * Function to be called when the review is published/saved as a draft.
    */
-  const onPublish = async () => {
+  const onSubmit = async (save = false) => {
```

Comment:
(nitpick) Could you make this parameter `publish` instead of save? (If it's not too much work)